docs/sources/administration/data-source-management/team_lbac.md

```diff
@@ -0,0 +1,137 @@
+
+For validating the rules, we recommend testing the rules in the Loki Explore view. This will allow you to see the logs that would be returned for the rule.
+
+#### Scenarios
+
+**Scenario 1: One rule setup for each team**
+
+We have two teams, Team A and Team B. Loki access is setup with `Admin` roles to have `Query` permission.
+
+- Team A has a rule `namespace="dev"`. A user that is part of Team A will have access to logs that match `namespace="dev"`.
+
+- Team B has a rule `namespace="prod"`. A user that is part of Team B will have access to logs that match `namespace="prod"`.
+
+**Scenario 2: One rule setup for a Team**
+
+We have two teams, Team A and Team B. Loki access is setup with `Editor`, `Viewer` roles to have `Query` permission.
+
+- Team A has a rule `namespace="dev"` configured. A user that is part of Team A will have access to logs that match `namespace="dev"`.
+
+- Team B does not have a rule configured for it. A user that is part of Team B, that is `Editor` or `Viewer` will have access to all logs (due to the query permission for the user).
+
+**Scenario 3: Multiple rules setup for one team**
+
+We have two teams, Team A and Team B. Loki access is setup with `Admin` roles having `Admin` permission.
+
+- Team A has rule `namespace="dev", namespace="prod"` configured. A user that is part of Team A will have access to logs that match `namespace="dev"` `AND` `namespace=prod`.
+
+- Team B has rule `namespace!="dev", namespace="prod"` configured. A user that is part of Team B will have access to logs that match `namespace!="bi"` `AND` `namespace=prod`.
+
+- A user that is part of Team A and Team B will have access to logs that match `(namespace="dev" AND namespace="prod") OR (namespace="dev" AND namespace!="bi")`.
+
+**Scenario 4: Two or more rules configured for a team**
+
+We have three teams, Team A, Team B and Team C. Team C is a new team and has no rules configured for it. Loki access is setup with `Editor` role having `Query` permission.
+
+Team A has a rule `namespace="dev"` configured. A user with `Editor` role has access to all logs. A user that is part of Team A without `Editor` role will get all logs that match `namespace="dev"`.
+
+Team B have two rules `namespace="dev"` and `namespace="prod"`. Team B, would be able to query all logs that match either `namespace="dev"` OR `namespace="prod"`.
+
+Team C have no rules. Team C has a user with an `Viewer`. Team C's `Editors`, will have access to query all logs.
+
+**Scenario 5: Rules that overlap**
+
+We have two teams, Team A and Team B.
+
+Team A has a rule `namespace="dev"`. A user in Team A will have access to logs that match `namespace="dev"`.
+
+Team B has a rule `namespace!="dev"`. A user in Team b will have access to logs that match `namespace!="dev"`.
+
+> _NOTE:_ A user that is part of Team A and Team B will have access to logs that match `namespace="dev"` `OR` `namespace!="dev"`.
```

Comment:
It might be good to say here that `namespace="dev"` `OR` `namespace!="dev"` are all the logs.